### PR TITLE
IEN 735 - add e2e test for report 1

### PR DESCRIPTION
**Pull request by coryatwater-FW**

### CODE REVIEW

**Review by sunghwan-park-fw — COMMENTED** [review]
How about defining a spec file for each report?

File: apps/api/test/reports.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+    app = moduleFixture.createNestApplication();
+    await app.init();
+  });
+
+  afterAll(async () => {
+    await app.close();
+  });
+
+  it('Add Applicant /ien (POST)', done => {
```
Comment (sunghwan-park-fw) [inline apps/api/test/reports.e2e-spec.ts:36]: `it` defines a test case. so, these two cases should be moved into the last one.

File: apps/api/test/applicant.e2e-spec.ts [hunk 2]
```diff
@@ -91,7 +91,7 @@ describe('ApplicantController (e2e)', () => {
       .get('/ien')
       .expect(res => {
         const { body } = res;
-        expect(body[1]).toBe(2);
+        expect([2, 4]).toContain(body[1]);
```
Comment (sunghwan-park-fw) [inline apps/api/test/applicant.e2e-spec.ts:94]: I wonder why body[1] could be 2 or 4

Comment (coryatwater-FW) [reply apps/api/test/applicant.e2e-spec.ts:94]: this is something I wanted to bring up in SU and discuss with you

**Review by sunghwan-park-fw — APPROVED** [review]

**Pull request merged**